code/modules/antagonists/hivemind/spells/hivemind_shock.dm

```diff
@@ -13,7 +13,7 @@
 		return
 	to_chat(user, "<span class='notice'>We begin increasing the psionic bandwidth between ourself and the vessel!</span>")
 	if(do_after(user,30,0,user))
-		if(target.mind.assigned_role in (GLOB.security_positions || GLOB.command_positions)) //Doesn't work on sec or command for balance reasons
+		if(target.mind.has_job(GLOB.security_positions + GLOB.command_positions)) //Doesn't work on sec or command for balance reasons
```

Comment:
this should be `|` or it will duplicate entries

Reply:
hmm actually nothing different, but yeah it looks weird